source/renderer/app/components/hardware-wallet/layouts/HardwareWalletWithNavigation.js

```diff
@@ -10,7 +10,7 @@ type Props = {
   children?: Node,
   activeItem: string,
   hasNotification?: boolean,
-  walletNotConnected: boolean,
+  walletNotConnected?: boolean,
```

Comment:
@DeeJayElly this should be the required parameter. Do not change  types just to make flow happy

Reply:
Fixed